ngraph_bridge/ngraph_builder.cc

```diff
@@ -2513,6 +2513,21 @@ static Status TranslateLogSoftmaxOp(
   return Status::OK();
 }
 
+static Status TranslateSoftplusOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_inp;
```

Comment:
https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/softplus.html

`log(exp(features) + 1)`

Reply:
May be add this - `log(exp(features) + 1)` as a comment for future reference.